charts/consul/templates/crd-peeringacceptors.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.connectInject.enabled .Values.global.peering.enabled }}
+{{- if .Values.controller.enabled }}
```

Comment:
Can this be reverted from this file and the one below it?

Reply:
+1

Reply:
@thisisnotashwin Thanks for taking the time to review.
I reverted the change to 
```
charts/consul/templates/crd-peeringacceptors.yaml
charts/consul/templates/crd-peeringdialers.yaml
```

And kept the change to 

```
	modified:   charts/consul/templates/crd-servicedefaults.yaml
	modified:   control-plane/api/v1alpha1/zz_generated.deepcopy.go
	modified:   control-plane/config/crd/bases/consul.hashicorp.com_servicedefaults.yaml
```

For file, control-plane/api/v1alpha1/zz_generated.deepcopy.go, not sure if the change looks good
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/84437068/200218950-4da166a6-6ce5-4870-9066-75335201cea0.png">

Could you help confirm if the changes are correct?

Reply:
Hey @erdanzhang !! Those changes look right!! I really appreciate you reaching out to confirm it!! Also, thank you so much, again, for your contribution!!